Nadim.CinemaReservationSystem.Web/ClientApp/src/DataAccess/CinemaDataAccess.js

```diff
@@ -113,10 +113,10 @@ const sendRequestToCreateCinemaRoom = (cinemaId, cinemaRoomInfo) =>{
 }
 
 const createCinemaRoomInfo = (name, cinemaRoomId) =>{
-    let cinemaRoom = {};
-    cinemaRoom.name = name;
-    cinemaRoom.cinemaRoomId = cinemaRoomId;
-    return cinemaRoom;
+    return {
```

Comment:
This method might be an overkill, constructing object in place might be just enough